src/main/java/com/syntaxphoenix/spigot/smoothtimber/config/config/CutterConfig.java

```diff
@@ -116,6 +119,9 @@ protected void onLoad() {
         ENABLE_EXCLUSION = check("exclusion.enabled", ENABLE_EXCLUSION);
         EXCLUDED_MATERIALS = check("exclusion.list", EXCLUDED_MATERIALS);
 
+        ENABLE_INCLUSION = check("inclusion.enabled", ENABLE_INCLUSION);
+        INCLUDED_MATERIALS = check("inclusion.list", INCLUDED_MATERIALS);
```

Comment:
A check for if this is actually a valid block should be added here (see [Material#isSolid](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/Material.html#isSolid()))

Reply:
I added the Material list safety with a warning in the console log.
Any invalid Material will be ignored.
But if a user enter an inexisting Material it will crash before in the check() function.

Reply:
I tested the plugin before creating the pull request and every thing was working fine. And it's still working fine.
I tested it with leaves and other solid blocks like stone.

Thank you for the checks and your swiftness